### ISSUE TITLE
Graduating a prerelease with no further changes

### ISSUE THREAD

**Issue by MartinBlackburn** [lerna/lerna#1675]
When we're happy with our latest prerelease we'd like to graduate the packages even if no further changes were made.
We thought this would be fixed in issue #1671, but we now get a new problem where lerna wont release an already released commit. 

## Expected Behavior
Version is updated from 0.0.1-prelease.0 to 0.0.1

## Current Behavior
```
lerna publish --force-publish=lobby --yes --conventional-commits --no-verify-registry --no-verify-access
lerna notice cli v3.3.2
lerna info versioning independent
lerna info Looking for changed packages since lobby@0.0.1-prerelease.0
lerna WARN force-publish lobby
lerna ERR! ERELEASED The current commit has already been released. Please make new commits before continuing.
```

## Steps to Reproduce (for bugs)
1. `git commit -m'chore: example change'`
2. `lerna version prerelease --force-publish=lobby --yes --preid=prerelease --npm-tag=prerelease` => `0.0.1-prerelease.0`
3. `lerna version --force-publish=lobby --yes --conventional-commits` => would like `0.0.1` got `The current commit has already been released`

<details><summary>lerna.json</summary><p>

```json
{
  "packages": [
    "packages/*",
    "apps/*"
  ],
  "command": {
    "publish": {
      "conventionalCommits": true,
      "message": "chore(release): publish",
      "allowBranch": "release/build"
    }
  },
  "npmClient": "yarn",
  "useWorkspaces": true,
  "version": "independent"
}
```
</p></details>

## Context
We prerelease packages on every merge to our release branch. After testing we will manually take the decision to graduate these packages.

## Your Environment

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.3.2 |
| `npm --version`  | 6.4.1 |
| `yarn --version` | 1.9.4 |
| `node --version` | v8.11.1 |

| OS | Version |
| --- | --- |
| macOS el Capitan | 10.11.6 |

**Comment by MartinBlackburn** [comment]
The current workaround I am using for this is to make an "empty" commit to the branch, such as whitespace change, then run `learna publish`
It does mean we are polluting the git history, but for now, it works.

**Comment by sergiohgz** [comment]
This is too close to our problem described in #1667, after updating to 3.3.2 we are getting that `ERELEASED` and blocked our workflow. Thanks @MartinBlackburn for your idea of an empty commit, it's making our git tree too ugly, but works for the meanwhile.

@evocateur what do you think about that `ERELEASED` error? We think that is not correct if we force

**Comment by x5a** [comment]
I would argue that the promotion of "staging" package > "latest" package should be allowed even _without_ force.

**Comment by evocateur** [comment]
1. It's really confusing when you title a new issue identically to a previously closed one.
2. Reproduction needed, a failing test would be ideal.

**Comment by akrantz** [comment]
I started using Lerna and ran into this same issue. I published using "lerna publish --dist-tag next" with a prerelease version (2.0.0-alpha.0). After validation that it was good, I wanted to re-publish the packages with the updated version using the "latest" dist-tag. But Lerna says that there are no changed. 

I ended up moving the "Publish" commit to a "next" branch in git and reset the master branch to the previous commit. I could then publish from the master branch because there were changed.

However, it would still be good to have a way to have Lerna be able to republish a prerelease version as a release version.
